src/components/badge/index.js

```diff
@@ -107,20 +112,58 @@ export default class Badge extends BaseComponent {
       width += borderWidth * 2;
       height += borderWidth * 2;
     }
-
     return {width, height};
   }
 
+  formatLabel() {
```

Comment:
just change the name to `getFormattedLabel`

Reply:
Done